ref/general/network-hardening.adoc

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description:
+:seo-title: Network security hardening
+:seo-description:
+:page-layout: general-reference
+:page-type: general
+= Network security hardening
+
+Network intrusions occur when unauthorized users gain access to network activity.
+With this access, they can alter traffic and steal network resources.
+
+// General questions
+// What are the most common examples of network intrusions or attacks? Would this be helpful in the intro?
+// What's the advice for where to put things in the cloud? Is this determined by the cloud or do you have to decide for yourself?
+// Is any of this information irrelevant in a cloud-native microservices context? 
+// Are there any gaps that I've missed?
+
+== Run behind firewalls
+Your production environment needs to run inside a firewall so your applications don't run in the DMZ.
```

Comment:
I think it might be good here to reference what DMZ standards for so maybe change "DMZ" to "DMZ (Demilitarized Zone)" on first reference. I know you explain what it is in the next sentence but having the term spelled out could help to make the concept even clearer.